src/main/java/Duke_Level4.java

```diff
@@ -0,0 +1,85 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+public class Duke_Level4 {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+        List<String> list = new ArrayList<>();
```

Comment:
Could this variable name be updated to show that there are multiple objects represented? 
```suggestion
        List<String> tasks = new ArrayList<>();
```